components/_global-navigation.scss

```diff
@@ -0,0 +1,123 @@
+$border-top-width: 6px;
+$nav-height: 49px;
+
+.wds-global-navigation {
+	border-bottom: 1px solid $color-faint-gray;
+	border-top: $border-top-width solid $color-slate-gray;
+	display: flex;
+	justify-content: center;
+
+	&::before, &::after {
+		content: "";
+		flex-grow: 1;
+	}
+
+	&::before {
+		border-top: $border-top-width solid $color-dark-blue-gray;
+		margin-top: -$border-top-width;
+	}
+
+	&__content-wrapper {
+		align-items: center;
+		display: flex;
+		height: $nav-height;
+		width: 1024px;
```

Comment:
I see one here: https://github.com/Wikia/design-system/blob/22833834db692ed4a3ec4cf4991eb48abc7dfec5/components/_global-footer.scss#L32
Although I don't think we should reuse it. These is one more `1024px` variable that defines breakpoint, but I'm not sure if they should be bound.